package.json

```diff
@@ -92,10 +92,10 @@
     "@wireapp/eslint-config": "2.0.0",
     "@wireapp/prettier-config": "0.5.0",
     "adm-zip": "0.5.9",
+    "archiver": "^5.3.1",
```

Comment:
Since we already have `zipjs` in the repo, can we use it for archive generation?